<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-V5MLE2I" failures="1" tests="2" name="Default test(failed)" time="0.115" errors="0" timestamp="2022-04-05T10:02:04 IST">
  <testcase classname="testngdiscussion.FailedTestCaseExecution" name="login" time="0.013">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at testngdiscussion.FailedTestCaseExecution.login(FailedTestCaseExecution.java:12)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase classname="testngdiscussion.FailedTestCaseExecution" name="dashBoard" time="0.0">
    <skipped/>
  </testcase> <!-- dashBoard -->
  <testcase classname="testngdiscussion.FailedTestCaseExecution" name="profile" time="0.0">
    <ignored/>
  </testcase> <!-- profile -->
  <testcase classname="testngdiscussion.FailedTestCaseExecution" name="logout" time="0.0">
    <ignored/>
  </testcase> <!-- logout -->
</testsuite> <!-- Default test(failed) -->
